browser/permissions/permission_manager_browsertest.cc

```diff
@@ -76,13 +76,9 @@ class PermissionManagerBrowserTest : public InProcessBrowserTest {
 
   ~PermissionManagerBrowserTest() override = default;
 
-  void SetUpCommandLine(base::CommandLine* command_line) override {
-    InProcessBrowserTest::SetUpCommandLine(command_line);
-    command_line->AppendSwitch(switches::kIgnoreCertificateErrors);
-  }
-
   void SetUpOnMainThread() override {
```

Comment:
and here `InProcessBrowserTest::SetUpOnMainThread();`

Reply:
added in https://github.com/brave/brave-core/commit/52c05a3594dbb439663b7b100cc719d80f3cb7db